lib/screens/home/home_bloc.dart

```diff
@@ -1,16 +1,18 @@
 import 'dart:async';
 
-import 'package:bloc/bloc.dart';
 import 'package:flutter/material.dart';
+import 'package:hydrated_bloc/hydrated_bloc.dart';
 import 'package:pub_client/pub_client.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 import 'package:tavern/screens/bloc.dart';
+import 'package:tavern/src/cache.dart';
+import 'package:tavern/src/repository.dart';
 
-class HomeBloc extends Bloc<HomeEvent, HomeState> {
+class HomeBloc extends HydratedBloc<HomeEvent, HomeState> {
   final PubHtmlParsingClient client;
-
   @override
   HomeState get initialState => InitialHomeState();
```

Comment:
since you refactored this to be a Hydrated Bloc you need to update this to:
```dart
@override
HomeState get initialState => super.initialState ?? InitialHomeState();
```
Otherwise the bloc won't used the cached state when it's available.

Reply:
Ok, got it. Done